src/chunk_insert_state.c

```diff
@@ -379,8 +379,11 @@ setup_on_conflict_state(ChunkInsertState *state, ChunkDispatch *dispatch, AttrNu
 
 		if (NULL == chunk_attnos)
 			chunk_attnos = convert_tuples_by_name_map(RelationGetDescr(chunk_rel),
-													  RelationGetDescr(first_rel),
-													  gettext_noop("could not convert row type"));
+													  RelationGetDescr(first_rel)
+#if PG13_LT
+                            , gettext_noop("could not convert row type")
```

Comment:
Is there are a good reason why we use `gettext` here at all? maybe just pass this string as is

Reply:
It's used to mark text for translation purposes.